opencga-client/src/main/python/pyopencga/opencga_client.py

```diff
@@ -130,23 +130,27 @@ def logout(self):
         for client in self.clients:
             client.token = self.token
 
-    def wait_for_job(self, response=None, study=None, job_id=None, retry_seconds=10):
+    def wait_for_job(self, response=None, study_id=None, job_id=None,
+                     retry_seconds=10):
         if response is not None:
-            study = response['studyUuid']
-            job_id = response['uuid']
+            study_id = response['result'][0]['study']['id']
+            job_id = response['result'][0]['uuid']
```

Comment:
is this correct? why do not just use jobId?

```suggestion
            job_id = response['result'][0]['id']
```